client/src/App.js

```diff
@@ -1,15 +1,15 @@
 import React from "react";
-import { BrowserRouter, Switch, Route } from "react-router-dom";
-import { Security, LoginCallback} from "@okta/okta-react";
+import { BrowserRouter, Route } from "react-router-dom";
+import { Security, LoginCallback } from "@okta/okta-react";
 
+import Home from "./components/views/home.component";
 import Navbar from "./components/layout/navbar.component";
-import TemplateDashboard from "./components/template/templateDashboard.component";
-import CreateInviteTemplate from "./components/template/createInviteTemplate.component";
 import SendInvite from "./components/invitation/sendInvite.component";
-import CreateUser from "./components/user/createUser.component";
 import UserProfile from "./components/user/userProfile.component";
-import Home from "./components/layout/home.component";
+import TemplateDashboard from "./components/views/templateDashboard.component";
+import CreateInviteTemplate from "./components/template/createInviteTemplate.component";
 
+// TODO make it a local variable or something
```

Comment:
Is it possible to remove comment, and use .env varaiables?

Reply:
No, this is client side